### ISSUE TITLE
Facebook login leaves "authData" undefined on new user object

### ISSUE THREAD

**Issue by ptsimpso** [ParsePlatform/Parse-SDK-iOS-OSX#843]
I am using PFFacebookUtilsV4 version 1.11.1, FBSDKCoreKit version 4.10.0, and FBSDKLoginKit version 4.10.0 all from cocoapods and using the most up to date Parse iOS SDK (1.12.0).

Within the last few days, new users signing up using the standard `PFFacebookUtils.logInInBackgroundWithReadPermissions(["email", "public_profile", "user_friends", "user_events"])` function are being created with the authData field undefined in the PFUser object. This makes it so that on subsequent login attempts (using the same method call), a new user is created instead of logging in the previously created PFUser object. This new user that is created once again has authData undefined.

Basically, logging in with the same Facebook credentials causes a new PFUser objects to be created each time with authData undefined.

**Comment by nlutsenko** [comment]
Hey @ptsimpso, this sounds very weird. Let's debug this together and find the reason behind it.
Can you share more details about your User? Is it logged in at the time of logging back in once again?
Are you using Anonymous users? Are you logging in the user with username/password?
All of these have impact on how the code behind logging a user behaves...

A basic flow description where you think the problem is happening would be great, a code snippet here would be ideal.

**Comment by ptsimpso** [comment]
Hi there,

Sorry I forgot to comment and close this out. It appears that the issue was due to a webhook we were using with the Parse JS SDK on the backend when a new user was created.
